consensus/service/src/tx_manager.rs

```diff
@@ -273,7 +275,7 @@ impl<E: ConsensusEnclaveProxy, UI: UntrustedInterfaces> TxManager<E, UI> {
         let tx_hashes: HashSet<&TxHash> = tx_hashes.iter().clone().collect();
         for tx_hash in tx_hashes {
             if let Some(entry) = cache.get(&tx_hash) {
-                tx_contexts.push(entry.context());
+                tx_contexts.push(entry.context().clone());
```

Comment:
I don't expect this to have dire performance implications, but just in case, have you slammed it? A bunch of changes went in lately and it'll be good to see no regressions are happening.

Reply:
I'm waiting on the deploy...

Reply:
I'm trying a different approach: CacheEntry now holds its context inside of an Arc, which should make this clone inexpensive.